scanners/sslyze.py

```diff
@@ -180,62 +180,91 @@ def parse_sslyze(xml):
 
         # Find the weakest available DH group size, if any are available.
         weakest_dh = 1234567890  # nonsense maximum
-        groups = target.select("acceptedCipherSuites cipherSuite keyExchange[Type=DH]")
-        for group in groups:
-            size = int(group["GroupSize"])
-            if size < weakest_dh:
-                weakest_dh = size
+        for cipher in accepted_ciphers:
+            if cipher.get('dh_info', None) is not None:
+                size = int(cipher['dh_info']['GroupSize'])
+                if size < weakest_dh:
+                    weakest_dh = size
 
         if weakest_dh == 1234567890:
             weakest_dh = None
 
         data['config']['weakest_dh'] = weakest_dh
 
     # If there was an exception parsing the certificate, catch it before fetching cert info.
-    if target.select_one("certinfo[exception]") is not None:
-        data['errors'] = target.select_one("certinfo[exception]")["exception"]
+    if False:
+        data['errors'] = "TODO"
```

Comment:
I'm not yet sure whether this will ever happen in the new sslyze code path. It'll become obvious in the output if it does!